README.md

```diff
@@ -88,12 +88,17 @@ exec:exec@destroy-pod`
 ### Run on Kubernetes/Openshift
 * See the [cryostat-operator](https://github.com/cryostatio/cryostat-operator)
 
-### Run on local podman
+### Run on local podman*
 * `run.sh`
 
-### Run on local podman with Grafana, jfr-datasource and demo application
+### Run on local podman with Grafana, jfr-datasource and demo application*
 * `smoketest.sh`
 
+*To run on local podman, [cgroups v2](https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html) must be enabled.
```

Comment:
Let's replace "must" with simply "should". It should still be possible to run with cgroups v1 if some user really needs that and doesn't necessarily care that they don't get rootless resource constraints with it.